react-native-gutenberg-bridge/index.js

```diff
@@ -52,4 +52,8 @@ export function onCapturePhotoPressed( callback ) {
 	return RNReactNativeGutenbergBridge.onCapturePhotoPressed( callback );
 }
 
+export function onImageQueryReattach( callback ) {
```

Comment:
Shouldn't we send up what is the ID of the media we want to reattach?

Reply:
It's not necessary to be passed here at all, as all we need to do is to tell the parent app which the new `callback` is so it can be set in [this line here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/505/files#diff-f2e1dfe800569dfb4ee4830f88779890R110).
Later the events are being broadcasted and there we tell the listeners which progress is being made for which `mediaId`.